### ISSUE TITLE
Make the default open app indicator indicate multiple windows

### ISSUE THREAD

**Issue by dvorapa** [micheleg/dash-to-dock#697]
Repoted downstream: https://github.com/Apricity-OS/apricity-build/issues/79

There are currently two ways to indicate app is running: default indicator (the small blue line under the icon) and dot indicator (which can show how many instances of the app are running)

I want to suggest to allow somehow (in preferences?) indicating multiple running instances by the default indicator (the small blue line under the icon). Maybe showing a hamburger ☰? or half-lines/little squares?

----
Extension version: 62
Gnome Shell version: 3.26.2
Linux distribution and version: Arch Linux latest/fully updated (build 20180201)
Whether you have a multi-monitor configuration: nope
Your settings:
```$ dconf dump /org/gnome/shell/extensions/dash-to-dock/
[/]
preferred-monitor=0
shift-click-action='minimize'
shift-middle-click-action='launch'
extend-height=true
dock-fixed=true
force-straight-corner=true
intellihide-mode='FOCUS_APPLICATION_WINDOWS'
show-windows-preview=false
show-show-apps-button=false
hot-keys=false
custom-background-color=false
custom-theme-customize-running-dots=true
apply-custom-theme=false
custom-theme-running-dots=true
unity-backlit-items=false
dock-position='LEFT'
custom-theme-shrink=true
icon-size-fixed=true
middle-click-action='launch'
background-color='#ffffff'
isolate-workspaces=true
```

**Comment by micheleg** [comment]
I believe the new indicator styles (not yet released but already in master [14514286b05b030017d4e49]) are already covering this need.

**Comment by dvorapa** [comment]
Cool, I'm looking forward to updating to the new release when done :)
